universal-login-sdk/lib/api/FutureWalletFactory.ts

```diff
@@ -1,24 +1,33 @@
 import {providers} from 'ethers';
```

Comment:
The new code in `FutureWalletFactory` is duplicated from `ExecutionFactory`. Will extract common code into base class in a follow-up PR, if that's OK.